packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -527,13 +528,6 @@
     }
 }
 
-.perseus-formats-tooltip {
```

Comment:
As these styles are really only relevant to the numeric input file now, I've added them over to a widget specific file.